### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by g0t00**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "cSpell.words": [
+    "addforeign",
+    "addother",
+    "addotherforeign",
+    "addpartial",
+    "addpartialforeign",
+    "Cooldown",
+    "editamount",
+    "fontawesome",
+    "fortawesome",
```
Comment (Nik-Sch) [inline .vscode/settings.json:11]:
```suggestion
```

File: client/esbuild.mjs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import * as esbuild from 'esbuild'
+import { argv } from 'process'
+
+const settings = {
+  entryPoints: ['src/index.tsx'],
+  bundle: true,
+  loader: {
+    '.woff': 'file',
```
Comment (Nik-Sch) [inline client/esbuild.mjs:8]: what are these woff files?

File: lib/add.ts [hunk 3]
```diff
@@ -134,21 +133,30 @@ async function add(ctx: MatchedContext<Context<Update>, "text">, useForeign: boo
 
   if (!message.text || messageText == '') {
     let replyObj = await ctx.reply(`Please enter description. @${message.from.username}`, {
-      reply_markup: {force_reply: true, selective: true}
+      reply_markup: { force_reply: true, selective: true }
     });
+    console.log('hi chat');
```
Comment (Nik-Sch) [inline lib/add.ts]: rm debug logs

File: lib/add.ts [hunk 4]
```diff
@@ -134,21 +133,30 @@ async function add(ctx: MatchedContext<Context<Update>, "text">, useForeign: boo
...
-      reply_markup: {force_reply: true, selective: true}
-    });
+    console.log('before try message');
+    try {
+      await app.bot.telegram.deleteMessage(replyObj.chat.id, replyObj.message_id);
+    } catch (err) {
+      console.log('caught delete message');
+      // do not care
+    }
+    replyObj = await app.bot.telegram.sendMessage(replyObj.chat.id, `Please enter amount. @${message.from.username}`,
```
Comment (Nik-Sch) [inline lib/add.ts]: why doesn't the normal `ctx.reply` work?

File: lib/app.ts [hunk 5]
```diff
@@ -60,22 +58,25 @@ class App {
...
-        }
-      });
-      this.bot.telegram.setWebhook('');
-
-    } else {
-      this.bot.launch();
-      console.log('starting Polling');
-      this.bot.telegram.setWebhook('');
-    }
+    // if (webHook) {
```
Comment (Nik-Sch) [inline lib/app.ts]: just remove it completely?

File: lib/app.ts [hunk 6]
```diff
@@ -91,7 +92,7 @@ class App {
     //   });
     // });
     this.bot.on('message', async (ctx, next) => {
-      // reply('mesesage');
+      // reply('message');
       // console.log(ctx);
```
Comment (Nik-Sch) [inline lib/app.ts]:
```suggestion

```

File: lib/group.ts [hunk 7]
```diff
@@ -1,19 +1,19 @@
...
-import { prop, post, DocumentType, arrayProp, pre, getModelForClass } from '@typegoose/typegoose';
-import { IGroupData, ITransaction, IMember, GroupState, IPayEntry } from './interfaces'
-const AsciiTable = require('ascii-table');
-import { v1 as uuid } from 'uuid';
+import { DocumentType, getModelForClass, post, pre, prop } from '@typegoose/typegoose';
 import * as _ from 'lodash';
-import { Member } from './member';
-import { Entry } from './entry';
+import * as moment from 'moment-timezone';
+import { v1 as uuid } from 'uuid';
```
Comment (Nik-Sch) [inline lib/group.ts]: uuid v1? default always seems to be 4 but maybe it doesn't matter xD

**Review by Nik-Sch — APPROVED** [review]

File: client/esbuild.mjs (hunk 2 above)
Comment (g0t00) [reply client/esbuild.mjs:8]: those are fonts

File: .vscode/settings.json (hunk 1 above)
Comment (g0t00) [reply .vscode/settings.json:11]: thats on purpose actually

**Pull request merged**